### PR TITLE
Update docs to new framework

### PR DESCRIPTION
**Pull request by aditya-manit**
Hi KingSuper here - I have updated docs to a new framework, which is far better than the previous one and much more customisable.

## Why the change was necessary ?

I noticed the previous documentation becoming a little untidy and not really clear, also notice the few highlighting issues like code wasn't properly visible in the tip section because of the green color. There was an indentation issue as well due to which it wasn't really clear whether a code snippet was under a point or under the article as a whole

Here are the few contrast comparison of old vs new

Old  | New
------------- | -------------
Screenshot 2021-12-06 at 2 24 59 AM | Screenshot 2021-12-06 at 2 27 20 AM
Screenshot 2021-12-06 at 2 29 27 AM | Screenshot 2021-12-06 at 2 30 22 AM

I have already installed necessary stuff and plugin and have customized the website to look nice and tidy. The process of adding the doc remains the same all we need to do different is not add an id on each page but to edit the `_sidebar.md` file, which is better than adding three line extra on each page just for id, its position and url.

## How to deploy ?

I believe we are still using vercel for automatic deployments and stuff - all we need to do is change a little config in vercel for this repo. Please change the deployment setting to what are shown below in the screenshot and we should be good to merge this PR.

Screenshot 2021-12-06 at 2 39 28 AM

## How to run in local ?

```bash
npm install -g docsify
docsify serve ./documentation/docs/
```

## Wanna see the site in action ?
Here is the deployed version of this PR: https://axelarate-community.vercel.app/#/

PS: The change might look big but its nothing except the new framework  and change needed in file due to it, file movements, fixing links and a little formatting etc

### CODE REVIEW

**Comment by aditya-manit** [comment]
Also Did I mention it has a custom cool looking "not found" page 😅 

Screenshot 2021-12-06 at 6 04 50 PM

File: documentation/docs/index.html [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+<script src="//unpkg.com/prismjs/components/prism-yaml.js"></script>
+<script src="//unpkg.com/prismjs/components/prism-log.js"></script>
+<script src="//unpkg.com/prismjs/components/prism-vim.min.js"></script>
+<script src="//unpkg.com/prismjs/components/prism-log.min.js"></script>
+<script src="//unpkg.com/prismjs/components/prism-toml.min.js"></script>
+<script src="//unpkg.com/docsify/lib/plugins/emoji.min.js"></script>
+<script src="//unpkg.com/docsify-copy-code"></script>
+<script src="https://cdn.jsdelivr.net/npm/docsify-example-panels"></script>
+<script src="//cdn.jsdelivr.net/npm/prismjs/components/prism-markdown.min.js"></script>
+<script src="//cdn.jsdelivr.net/npm/docsify-sidebar-collapse/dist/docsify-sidebar-collapse.min.js"></script>
```
Comment (ggutoski) [inline documentation/docs/index.html:72]: Not sure about all this script injection.  Is this necessary?  @canhtrinh

File: documentation/docs/parent-pages/exercises.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# Testnet Exercises
+-----
+
+<!-- panels:start -->
+
+<!-- div:right-panel -->
+   <center>
+  <a href="../#/Exercises/exercise-4">
+<img src="https://i.ibb.co/dBGGTFR/cropped.jpg"
```
Comment (ggutoski) [inline documentation/docs/parent-pages/exercises.md:9]: Not sure whether we prefer to host all assets locally.  I guess it's okay for now.

**Review by ggutoski — APPROVED** [review]
This is great!  Thank you!  Just a couple comments.  I'll wait a short while before merging in case others have feedback.

File: documentation/docs/index.html (hunk 1 above)
Comment (aditya-manit) [reply documentation/docs/index.html:72]: Hi @ggutoski so each of these script adds a functionality - to give you an example:
```
<script src="//cdn.jsdelivr.net/npm/docsify/lib/plugins/search.min.js"></script>
```
provides the search fucntionality
```
<script src="//unpkg.com/prismjs/components/prism-*.min.js"></script>
```
anything starting with `prism` provide language highlighting - which I have added to highlight the code

```
<script src="//unpkg.com/docsify-copy-code"></script>
```
this allows a code block to be each copied by `copy to clipboard` button

```
<script src="//unpkg.com/docsify/lib/plugins/emoji.min.js"></script>
```
this turns `:bulb:` into 💡  and other emojis 
```
<script src="//cdn.jsdelivr.net/npm/docsify-sidebar-collapse/dist/docsify-sidebar-collapse.min.js"></script>
```
this allows a dropdown in sidebar and etc etc

File: documentation/docs/parent-pages/exercises.md (hunk 2 above)
Comment (aditya-manit) [reply documentation/docs/parent-pages/exercises.md:9]: ahh makes sense - I got a little lazy there 😛 

From what I have observed I think it caches some stuff in local browser cache too (not really sure though) - so these assets will be fetch once and then would be easily loaded for every click after that.

But yeah, feel free to ask me - I can definitely move all the hosted images to a local `_media` or `_ assets` folder  in this or any of the following PRs

Comment (aditya-manit) [reply documentation/docs/parent-pages/exercises.md]: okay so this is done - all hosted images should be now in `_media` directory

**Pull request merged**